vendor/github.com/opencontainers/runtime-spec/specs-go/version.go

```diff
@@ -11,7 +11,7 @@ const (
 	VersionPatch = 0
 
 	// VersionDev indicates development branch. Releases will be empty string.
-	VersionDev = ""
+	VersionDev = "+dev"
```

Comment:
prefer to move up to a tagged non-dev release version of runc if possible @thaJeztah @AkihiroSuda thoughts?

Reply:
LGTM otherwise just need to settle on what is a good version of runc to promote to.. ATM

Reply:
This is the runtime-spec, not runc. We are using a tagged version of runc.

As for tagged versions of the _runtime-spec_, it is very hard touse a tagged version. For example, since we moved to go modules (using go.mod), we just use a tagged version in the last few months, when there was a release. Otherwise, we always used an unreleased version. The same happens on runc, we used a tagged version for a short time, runc main is not using a tagged version now, and the last release was not so long ago.

Reply:
nod.. I saw mount extensions and figured the use of.. would result in a new min version of runc to have included this version of the runtime spec... should've said the runc that will include the use of this dev version of runtime spec and if they will be waiting for the spec to pop another release. 

thinking ahead for if this user ns work would be cherry picked to containerd v1.6/7 and what version of runc is needed

Reply:
runc 1.2 is needed (whenever that is released), and it will use this same version of the runtime-spec (for sure not older than this).

Using a dev version of the runtime spec is very common, in fact containerd 1.6 is doing that already: https://github.com/containerd/containerd/blob/release/1.6/go.mod#L48

Reply:
But we detect at runtime if the runc version supports idmap mounts, if it is not supported and a container requests idmap mounts, we fail with a clear error.

Reply:
ok.. can we show the log out of it working on a private build...